als-common/shared/src/main/scala/org/mulesoft/als/configuration/MaxSizeException.scala

```diff
@@ -0,0 +1,6 @@
+package org.mulesoft.als.configuration
+
+import amf.core.client.scala.exception.AmfUnhandledException
+
+case class MaxSizeException(url: String, maxSize: Int)
+    extends AmfUnhandledException(s"maximum file size of $maxSize exceeded")
```

Comment:
Whats is the unit of $maxSize? I mean Mb, Kb, characters...

Reply:
those are bytes